notebooks/02_analyses/hatesonar_analysis.ipynb

```diff
@@ -0,0 +1,1416 @@
+{
```

Comment:
<p>Shouldn't this just be 3 columns? I did not see where the hatesonar calculations occurred above? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/mailing-list-analysis-toolkit/pull/54/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>The set returned by hatesonar is (top match, Hate speech value, Discriminatory text value, and neither value) </p>

Reply:
<p>Yes, but you are applying the hatesonar function below, so the output above is a little confusing as its unclear where these additional columns come from. This notebooks appears to be have run out of order. If you run it in order the output of the above cell only has 3 columns. Could you re-run the notebook from top to bottom so that the correct outputs are being displayed? </p><p> </p>